en/publish/publish-oppo-instant-games.md

```diff
@@ -1,125 +1,132 @@
+Starting with __v2.0.5__, __Cocos Creator__ officially supports the release of games to the __OPPO Mini Game__ platform. It takes just a single click with __Cocos Creator__ to publish to the __OPPO Mini Game__ platform.
 
-## 环境配置
+## Environment Configuration
 
-- 下载 [快游戏调试器](http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//bbce19cecbf7c26c396762d61192a11e.zip)，并安装到 Android 设备上（建议 Android Phone 6.0 或以上版本）
+- Download [Quick Game Debugger] (http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//bbce19cecbf7c26c396762d61192a11e.zip) and install it on your Android device (Android Phone 6.0 or above is recommended)
 
-- 全局安装 [nodejs-8.1.4](https://nodejs.org/zh-cn/download/) 或以上版本
+- Global installation [nodejs-8.1.4] (https://nodejs.org/zh-cn/download/) or above
 
-- 下载 [quickgame-toolkit](http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//0974ab43dc361f148189515ee254ff6d.zip)，并解压。
+- Download [quickgame-toolkit] (http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//0974ab43dc361f148189515ee254ff6d.zip) and unzip it.
 
-  如果想要减少第一次使用 Cocos Creator 构建 rpk 的等待时间，可以执行以下操作先安装项目依赖：
+  If you want to reduce the wait time for building `rpk` with __Cocos Creator__ for the first time, you can do the following to install the project dependencies:
 
   ```bash
-   # 命令行指定到解压后的 quickgame-toolkit 目录下
+   # Command line is specified in the unpacked quickgame-toolkit directory
    cd quickgame-toolkit
-   # 安装项目的依赖
+
+   #Installation project dependencies
    npm install
   ```
 
-- 根据用户自己的开发需求判断是否需要安装 [调试工具](http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//6de9d1c3f06030ae7c52f5105f60383f.zip)。
+- Determine whether you need to install [Debugging Tools] (http://cdofs.oppomobile.com/cdo-activity/static/quickgame/tools//6de9d1c3f06030ae7c52f5105f60383f.zip) according to your own development needs.
 
-## 发布流程
+## Release process
 
-一、使用 Cocos Creator 打开需要发布的项目工程，在 **构建发布** 面板的 **发布平台** 中选择 **OPPO Mini Game**。
+1. Use __Cocos Creator__ to open the project project that needs to be released. Select **OPPO Mini Game** in the **Publishing Platform** dropdown of the **Build Release** panel.
 
 ![](./publish-oppo-instant-games/build_option.jpg)
 
-其中 **游戏包名**、**游戏名称**、**桌面图标**、**游戏版本名称**、**游戏版本号**、**平台版本号**、**本地 npm 安装路径**、**本地 quickgame-toolkit 路径** 这些参数为必填项，根据用户的需求及参数输入框的提示信息进行填写。而 **小包模式**、**小包模式服务器路径**、**构建发布程序包** 为选填项。相关参数配置具体的填写规则如下：
+Where **game package name**, **game name**,**desktop icon**,**game version name**,**game version number**,**platform version number**,**local Npm installation path**, **local quickgame-toolkit path** These parameters are required and filled in according to the user's requirements and the prompt information in the parameter input box. The **packet mode**, **packet mode server path**, **build release package** is optional. The specific filling rules for the relevant parameter configuration are as follows:
```

Comment:
1. Spaces are missing after the ","
2. The first letter of the parameter name needs to be capitalized

**Game Package Name**, **Game Name**, **Desktop Icon**, **Game Version Name**, **Game Version Number**, **Platform Version Number**, **Local Npm Installation Path**, **Local quickgame-toolkit Path**

**Packet Mode Server Path**, **Build Release Package**

Reply:
fixed.